proprietary/Components/src/denhaag/hero.tokens.json

```diff
@@ -171,6 +171,41 @@
             "value": "16rem"
           }
         }
+      },
+      "theme-page": {
+        "height": { "value": "288px" },
```

Comment:
In NLDS TheHague they use rems instead of pixels, please continue to to use rems :)